### ISSUE TITLE
Right-click in the List loses selection of multiple points

### ISSUE THREAD

**Issue by dfoll** [Esri/coordinate-conversion-addin-dotnet#584]
### Current behavior
If you select multiple coordinates in the list, then right-click on **one of the selected coordinates**, (the current multi-selection is lost) click Copy, then paste into a text document, you only get the single coordinate that you actually clicked.  

### Expected behavior
If you select multiple coordinates in the list, then right-click on one of the selected coordinates, click Copy, then paste into a text document, all of the points that were selected should be copied over.

Windows standard behavior would be to get the other selected items if one item of the selected group is right-click/copied. For example, I have files A, B, C, D.  I multiple-select B and D, then right-click file D and click Copy.
In a new location, I paste.  Both file B and D are copied to the new location.
![image](https://user-images.githubusercontent.com/3801392/54233047-7e134300-44e2-11e9-8ef7-a12001511c37.png)
Currently it is treating the case of I have B and D selected, and I right-click A and click Copy. In that case, only A should be copied.

If you make a multiple selection and use Ctrl+C to copy, you get the multiple selection.
![image](https://user-images.githubusercontent.com/3801392/54232678-9898ec80-44e1-11e9-926a-91707f2ab82b.png)

Multiple select, right-click, copy/paste retains all of the selected features in the released version of CC, so this is a new issue with the test build I'm using: CTLocalGovTeam-dev\57

### Notes
- This behavior was specifically implemented to address a problem that was seen with #569 where when you had to select a point then right-click it to view details. 
- This will impact the behavior to right-click > View details. (see #585)

**Comment by dfoll** [comment]
Testing using CC build 94 with Pro 2.4. Verified you can select multiple points and right-click to copy those selected points without losing the selection.

![image](https://user-images.githubusercontent.com/5322669/58206514-a57e4d00-7c95-11e9-8352-3fd21df24126.png)

**Comment by dfoll** [comment]
### Doc updates:
- ...\Solutions\DefenseSolutions\CoordinateConversion\Testing\csm-build-brokey-broke-on-10-4-1\107
- _MT build is currently broken, so no pdf there_
- https://solutionsdev.arcgis.com/defense/help/military-tools/resources/june-2019-military-tools-for-arcgis.htm
- https://solutionsdev.arcgis.com/defense/help/coordinate-conversion/resources/june-2019-coordinate-conversion.htm

**Comment by dfoll** [comment]
@lfunkhouser confirmed fixed on 10.4.1. do you want to wait on the verified label for G-QC (Doc)?

**Comment by lfunkhouser** [comment]
Tested from ...\Solutions\DefenseSolutions\MilitaryToolsForArcGIS\Testing\csm-update-build-to-10-4-1-\275. Verified issue 584 added to issues addressed. Verified functionality on 10.7.1 (11595) and 2.4 (19822). 
Doc updates needed:
- [ ] Update Folder names for ArcGIS Pro (2.2-2.4) and ArcMap (10.4.1-10.7.1) in Contents https://solutionsdev.arcgis.com/defense/help/military-tools/resources/contents.htm
- [ ] Update Folder names for ArcGIS Pro (2.2-2.4) and ArcMap (10.4.1-10.7.1) in Contents https://solutionsdev.arcgis.com/defense/help/coordinate-conversion/resources/contents-for-coordinate-conversion.htm
Jenkins script updates needed:
- [ ] Update add-in version Military Tools to 3.5.0.build#
- [ ] Update add-in version Military Tools ArcMap to 3.5.0.build#

**Comment by dfoll** [comment]
@lfunkhouser ...noted. i believe @csmoore is still in progress on troubleshooting the builds... we will need that to be wrapped up to do the verification on these four checkboxes

**Comment by dfoll** [comment]
UPDATED: 
- https://solutionsdev.arcgis.com/defense/help/military-tools/resources/contents.htm
- ...\review_pdf\military_tools_release_notes

**Comment by dfoll** [comment]
MT are now 3.5.0.build
CC versions are 4.3.1.126 (Pro) 2.4.1.126 (AM)

**Comment by dfoll** [comment]
I do not understand what the G - Doc (Github) label is for
